cli/flox/src/utils/message.rs

```diff
@@ -30,3 +32,10 @@ pub(crate) fn updated(v: impl Display) {
 pub(crate) fn warning(v: impl Display) {
     print_message(std::format_args!("⚠️  {v}"));
 }
+
+pub(crate) fn package_installed(pkg: &PackageToInstall, environment_description: &str) {
+    updated(format!(
+        "'{}' installed to environment {environment_description}",
+        pkg.id
+    ));
+}
```

Comment:
nit(nb): this is a bit too specific in this module.
I think its better for this to be in the context, even if that's the same (or similar) message twice

Reply:
I thought it was worth factoring out to enforce consistent wording. I agree it feels a little weird in message, and we can move it to `environment` at some point especially if we add more environment functions, but I figured it's most likely to be found here at the moment